components/cam/src/physics/crm/crm_input_module.F90

```diff
@@ -1,5 +1,5 @@
 module crm_input_module
-   use params, only: crm_rknd
+   use params_kind, only: crm_rknd
 #ifdef MODAL_AERO
    use modal_aero_data, only: ntot_amode
 #endif
```

Comment:
File moved from `.../crm/sam/...` to `.../crm/...`. This also needs a path update to `crm_input_module.F90` in `Depends.pgigpu.cmake`.

Same for the other four: i.e.
```
diff --git a/cime_config/machines/Depends.pgigpu.cmake b/cime_config/machines/Depends.pgigpu.cmake
index 6496cb012e..f84c231247 100644
--- a/cime_config/machines/Depends.pgigpu.cmake
+++ b/cime_config/machines/Depends.pgigpu.cmake
@@ -41,11 +41,11 @@ set(FILES_NEED_OPENACC_FLAGS
   cam/src/physics/crm/sam/coriolis.F90
   cam/src/physics/crm/sam/crmtracers.F90
   cam/src/physics/crm/sam/crm_ecpp_output_module.F90
-  cam/src/physics/crm/sam/crm_input_module.F90
+  cam/src/physics/crm/crm_input_module.F90
   cam/src/physics/crm/sam/crm_surface.F90
-  cam/src/physics/crm/sam/crm_output_module.F90
-  cam/src/physics/crm/sam/crm_rad_module.F90
-  cam/src/physics/crm/sam/crm_state_module.F90
+  cam/src/physics/crm/crm_output_module.F90
+  cam/src/physics/crm/crm_rad_module.F90
+  cam/src/physics/crm/crm_state_module.F90
   cam/src/physics/crm/sam/damping.F90
   cam/src/physics/crm/sam/grid.F90
   cam/src/physics/crm/sam/diagnose.F90
@@ -75,7 +75,7 @@ set(FILES_NEED_OPENACC_FLAGS
   cam/src/physics/crm/sam/uvw.F90
   cam/src/physics/crm/sam/vars.F90
   cam/src/physics/crm/sam/zero.F90
-  cam/src/physics/crm/sam/openacc_utils.F90
+  cam/src/physics/crm/openacc_utils.F90
   cam/src/physics/crm/sam/sat.F90 )

 # add accelerator/gpu flags for MPAS files
```

Reply:
I think we're considering the pgigpu configuration "dead" at this point because it's often slower than the CPU due to the pool allocator issue. Especially since SAM++ is working so well, I doubt we'll ever use the openACC code again, but we should wait a bit to make that decision, or allow @xyuan to remove it as part of his work with openmp.